imports/ui/Gauge.jsx

```diff
@@ -55,15 +55,16 @@ export default class Gauge extends Component{
     return (
 
         <div className="row noLeftRightMargin gauge-component-container">
-          <div className={"col-md-6 no-side-padding gauge-text " + this.gauge_text_padding}>
-            <h1 className={this.gauge_text_top}>
-                {this.state.rating}
-            </h1>
+          <div className={"col-md-6 col-sm-6 col-xs-6 no-side-padding gauge-text-container " + this.gauge_text_padding}>
+            <p className="gauge-text">
+                <span className={this.gauge_text_top}>{this.state.rating}</span>
+                <span className="gauge-text-bottom ">{this.props.text}</span>
+            </p>
             <h1 className="gauge-text-bottom">
-              {this.props.text}
```

Comment:
Why this change?

Reply:
- `h1` --> `p` was just a preference

- putting both into the same `p` tag surrounded by their own `span` tags was for vertical centering using auto margins to work

Reply:
Then why do we left an empty h1 here?

Reply:
Ahh totally missed this `h1` tag, will remove now, thanks for the catch!